actions/setup-conftest/action.yaml

```diff
@@ -0,0 +1,51 @@
+name: Install conftest
+description: Install conftest
+
+inputs:
+  version:
+    description: |
+      Version of conftest to install.
+    default: 0.55.0
```

Comment:
suggestion: could default to `latest` like the `fetch-gh-release-asset` action does, and this could become optional then

Reply:
Latest would need more plumbing to be able to cache, I'd prefer hardcoding it now.

Reply:
no worries 👍